subprojects/ide-native/src/integTest/groovy/org/gradle/ide/visualstudio/VisualStudioMultiProjectIntegrationTest.groovy

```diff
@@ -321,9 +427,37 @@ class VisualStudioMultiProjectIntegrationTest extends AbstractInstalledToolChain
         greetLibProject.projectConfigurations['debug'].includePath == filePath("src/greetings/headers")
     }
 
+    @IgnoreIf({GradleContextualExecuter.daemon})
```

Comment:
It's not clear to me why this is ignored with the daemon executor - might benefit from a comment.

Reply:
Good point, I will add a comment here. We should also think about adding test coverage to the generated command line for both Xcode and VS.

Reply:
Done.